### ISSUE TITLE
Thumbnails rights problem

### ISSUE THREAD

**Issue by labr1005** [modxcms/pThumb#2]
When I set the system setting "new_file_permissions" to "644" pThumbs creates thumbnails that have rights "-w----r--" or octal "204" - so they can not be accessed.

**Comment by oo12** [comment]
Thanks for reporting this. I'll fix in the next version, most likely this weekend.
For now try changing [this line](https://github.com/oo12/phpThumbOf/blob/pThumb/core/components/phpthumbof/model/phpthumbof.class.php#L58) from:

```
$this->config['newFilePermissions'] = $modx->getOption('new_file_permissions', $options, 0664);
```

to:

```
$this->config['newFilePermissions'] = octdec($modx->getOption('new_file_permissions', $options, '0664'));
```

**Comment by oo12** [comment]
Fixed in 2.0.0-pl.
